omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/converter/openmrs2_2/CodedOrFreeTextConverter.java

```diff
@@ -56,7 +56,6 @@ public CodedOrFreeText getByUniqueId(String uuid) {
 		return null;
 	}
 	
-	@Override
```

Comment:
Did you intentionally remove the above?

Reply:
It wasn't intentional. This happened while I was solving a merge conflict. Am adding it now.